File: 217-contains-duplicate/217-contains-duplicate.kt
class Solution {
    fun containsDuplicate(nums: IntArray): Boolean {
    var ok:Boolean
    var mp = HashMap<Int, Int>()
    for (i in 0 until nums.size) {
        var count = mp[nums[i]]
        if (count == null) {
            count = 0
        }
        mp[nums[i]] = count + 1
    }
    if(mp.size==nums.size){
        ok=false
    }
    else{
        ok=true
    }
    return ok
    }
}

File: 217-contains-duplicate/217-contains-duplicate.cpp
class Solution {
public:
    bool containsDuplicate(vector<int>& v) {
        bool ok = false;
		sort(v.begin(), v.end());
		for (int i = 0;i < v.size() - 1;i++) {
			if (v[i] == v[i + 1]) {
				ok = true;break;
			}
		}
        return ok;
    }
};